larrecodnn/ImagePatternAlgs/Tensorflow/Modules/EvaluateROIEff_module.cc

```diff
@@ -0,0 +1,151 @@
+////////////////////////////////////////////////////////////////////////
+// Class:       EvaluateROIEff
+// Plugin Type: analyzer (art v3_05_00)
+// File:        EvaluateROIEff_module.cc
+//
+// Generated at Sun May  3 23:16:14 2020 by Tingjun Yang using cetskelgen
+// from cetlib version v3_10_00.
+////////////////////////////////////////////////////////////////////////
+
+#include "art/Framework/Core/EDAnalyzer.h"
+#include "art/Framework/Core/ModuleMacros.h"
+#include "art/Framework/Principal/Event.h"
+#include "art/Framework/Principal/Handle.h"
+#include "art/Framework/Principal/Run.h"
+#include "art/Framework/Principal/SubRun.h"
```

Comment:
Recommend getting rid of these header files.

Reply:
Done.